pandas/io/clipboard.py

```diff
@@ -86,6 +89,12 @@ def to_clipboard(obj, excel=None, sep=None, **kwargs):  # pragma: no cover
             obj.to_csv(buf, sep=sep, **kwargs)
             clipboard_set(buf.getvalue())
             return
+        except UnicodeEncodeError:
+            # try again with encoding from locale
+            from locale import getdefaultlocale
+            obj.to_csv(buf, sep=sep, encoding=getdefaultlocale()[1], **kwargs)
```

Comment:
I believe using default locale is not necessary. `utf-8` should do the job (plus decoding `buf.getvalue` to `utf-8`) . The windows code of `pyperclip` (`pandas.util.clipboard`) would ultimately convert it back to unicode.